pkg/controller/sparkapplication/controller.go

```diff
@@ -292,19 +297,6 @@ func (c *Controller) getAndUpdateAppState(app *v1beta1.SparkApplication) error {
 				currentDriverState.completionTime = metav1.Now()
 			}
```

Comment:
One more thing, you can add a break here since there should only be one driver pod.

Reply:
Yep. I should do that. Thanks for reminding me.